charts/authentik/values.yaml

```diff
@@ -604,6 +604,46 @@ server:
     # -- uses `server.service.servicePortHttps` instead of `server.service.servicePortHttp`
     https: false
 
+  route:
+    main:
+      # -- enable an HTTPRoute resource for the authentik server.
+      # Be aware that this is an early beta of this feature. We don't guarantee this works and is subject to change.
+      enabled: false
+      # -- Set the route apiVersion
+      apiVersion: gateway.networking.k8s.io/v1
+      # -- Set the route kind
+      kind: HTTPRoute
+
+      # -- Route annotations
+      annotations: {}
+      # -- Route labels
+      labels: {}
+
+      # -- Route hostnames
+      hostnames: []
+      # -- Reference to parent gateways
+      parentRefs: []
+
+      # -- Create http route for redirect (https://gateway-api.sigs.k8s.io/guides/http-redirect-rewrite/#http-to-https-redirects).
+      # Take care that you only enable this on the http listener of the gateway to avoid an infinite redirect.
+      # Matches, filters and additionalRules will be ignored if this is set to true
+      httpsRedirect: false
+
+      # -- uses `server.service.servicePortHttps` instead of `server.service.servicePortHttp`
+      https: false
+
+      # -- Route matches
+      matches:
+        - path:
+            type: PathPrefix
+            value: /
```

Comment:
On the Ingreess, the path references `{{ .Values.authentik.web.path }}` so here probably should also